src/engraving/playback/mapping/stringssetupdataresolver.cpp

```diff
@@ -267,6 +267,14 @@ PlaybackSetupData StringsSetupDataResolver::doResolve(const Instrument* instrume
                                                                                 SoundSubCategory::Plucked } } },
         { "bowed-synth", { SoundId::Synthesizer, SoundCategory::Strings, { SoundSubCategory::Electric,
                                                                            SoundSubCategory::Bowed } } },
+        { "timple-canario", { SoundId::TimpleCanario, SoundCategory::Strings, { SoundSubCategory::Acoustic,
```

Comment:
You need to add these new IDs (new instrument definitions) to instruments.xml

Reply:
@shoogle FYI

Reply:
Done. `share/instruments/instrumentsxml.h` is also modified.